### PR TITLE
integration-tests: network interface

### PR DESCRIPTION
**Pull request by fgimenez**
The tests exercise these aspects of the interface:
* autoconnect
* disconnect and reconnect
* disabled network access after disconnect

### CODE REVIEW

File: integration-tests/tests/interfaces_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/data"
+
+	"gopkg.in/check.v1"
+)
+
+var _ = check.Suite(&networkInterfaceSuite{})
+
+const (
+	connectedPattern = "(?msi)" +
+		"slot +plug\n" +
```
Comment (zyga) [inline integration-tests/tests/interfaces_test.go]: This was changed to "Slot Plug"

File: integration-tests/tests/interfaces_test.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		".*" +
+		"^:network +network-consumer\n" +
+		".*"
+	disconnectedPattern = "(?msi)" +
+		"slot +plug\n" +
```
Comment (zyga) [inline integration-tests/tests/interfaces_test.go]: Ditto

**Comment by zyga** [comment]
Just one trivial. LGTM

**Comment by fgimenez** [comment]
@zyga thanks a lot for your review! :) fixed capitalization and removed regexp `ì` flag

**Comment by fgimenez** [comment]
retest this please

**Comment by fgimenez** [comment]
retest this please

**Comment by niemeyer** [comment]
Nice! LGTM assuming the integration tests actually pass for this branch.

**Comment by fgimenez** [comment]
@niemeyer thanks for your review, they are passing locally, once #1065 lands I'll add those changes and ask for a retest to (hopefully) see them passing here too :)

**Comment by fgimenez** [comment]
all green after a rebase, merging

**Pull request merged**